includes/admin/class-wc-stripe-rest-payment-intents-controller.php

```
@@ -0,0 +1,220 @@
+	 */
+	public static function sanitize_created_field( $value, WP_REST_Request $request, string $param ) {
+		if ( ! is_array( $value ) ) {
+			$value = sanitize_text_field( $value );
+		} else {
+			foreach ( $value as $operator => $operand ) {
+				$value[ $operator ] = sanitize_text_field( $operand );
+			}
+		}
+
+		return $value;
+	}
```

PHPMD (2.15.0)

[error] The method sanitize_created_field is not named in camelCase. (undefined)

(CamelCaseMethodName)